session.go

```diff
@@ -679,6 +689,9 @@ type ReadPreference struct {
 	// Mode determines the consistency of results. See Session.SetMode.
 	Mode Mode
 
+	// MaxStalenessSeconds specify a maximum replication lag, or “staleness” in seconds, for reads from secondaries.
+	MaxStalenessSeconds int
```

Comment:
Is this missing an `omitempty` tag? I imagine you don't want to send `maxStalenessSeconds: 0` with every request if it was not set by the user.

Reply:
hi @domodwyer, the wire information is sent by socket.go(line: 131), I suppose that tag is not necessary here, thanks.